### DISCUSSION TITLE
I need to change only the style of selected component in grapesjs

### DISCUSSION THREAD

**Discussion by champion5star** [GrapesJS/grapesjs#5551] [Q&A]
I made custom component and I add several ones in editor.
By the way, when I select one component and change style, the styles of others are changing together.

https://jumwdwt3xc.vmaker.com/record/ryyqxMoUrQgopia7
Please check this video and help me.

**Comment by artf** [comment]
I guess you need this option https://grapesjs.com/docs/modules/Selectors.html#component-first-selectors
